test/MSVC/pch_gen/fixture/SConstruct

```diff
@@ -0,0 +1,19 @@
+import os
+DefaultEnvironment(tools=[])
+# TODO:  this is order-dependent (putting 'mssdk' second or third breaks),
+# and ideally we shouldn't need to specify the tools= list anyway.
+env = Environment(tools=['mssdk', 'msvc', 'mslink'])
+env.Append(CPPPATH=os.environ.get('INCLUDE', ''),
```

Comment:
why propagating os.environ here?

Reply:
this test was mostly copied from the msvc.py test, with some minor modifications to test pch generator.